### ISSUE TITLE
Populating Embedded Bin with JSON

### ISSUE THREAD

**Issue by donundeen** [jsbin/jsbin#2137]
I'm interested in embedding JSBin in a Javascript-based web app, in such a way that the JSBin has access to JSON data, and functions, created elsewhere on the page.

Say I have some html like this:

```
var foo = {a: "123" , x:"foo"};
```

Then,  I embed the JSBin like this:
<a class="jsbin-embed" href="http://localhost:3100/liv/latest/embed?html,js,output">JS Bin....</a><script src="http://localhost:3100/js/embed.js">
</script>

Within the javascript tab of the JSBin, I'd like to be able to access
foo.a;

Basically what I'm doing is creating an environment that lets users mix and match various web service calls with assorted GUI widgets. I'd like to add a widget that lets them play with js/html/css within the page, using the data loaded in other widgets. JSBin would be a good fit for this, but I don't want to require them to call the data via ajax from within the JSBin, I want to be inserted into the JSBIN javascript scope.

Inserting custom functions into the scope would be great as well.

Is that possible?
